routes/api.mjs

```diff
@@ -65,17 +64,17 @@ router.post("/popularPicks/", async (req, res) => {
       res.json({ error: "missing location data cookie, set location first." })
         .end;
     }
-    const { data, responseCookies } = await popularRestaurants(
-      req.body,
-      requestCookies
-    );
-    res.setHeader("Set-Cookie", responseCookies);
+    const { data } = await popularRestaurants(req.body, requestCookies);
```

Comment:
After we change the call in `api/get.mjs` we can change this line to

`res.json(await popularRestaurants(req.body, requestCookies));`